cornichon-core/src/main/scala/com/github/agourlay/cornichon/dsl/ProvidedInstances.scala

```diff
@@ -1,15 +0,0 @@
-package com.github.agourlay.cornichon.dsl
-
-import cats.instances._
-
-// Importing by default Cats instances for common types to make it easier for potential non dev-users.
-// Built-in DSL/assertions work with Show, Eq and Order
```

Comment:
This trait was provided to ease the usage of the DSL.
It is going to be interesting to see how this affects compile time on big project.